docs/admin-get-started.md

```diff
@@ -9,102 +9,245 @@ import Tabs from '@theme/Tabs';
 
-- Java admin API
+````mdx-code-block
+<Tabs groupId="api-choice"
+  defaultValue="pulsar-admin"
+  values={[{"label":"pulsar-admin","value":"pulsar-admin"},{"label":"REST API","value":"REST API"},{"label":"Java","value":"Java"}]}>
+<TabItem value="pulsar-admin">
+
+[pulsar-admin CLI](pathname:///reference/#/@pulsar:version_reference@/pulsar-admin/): it’s a command-line tool and is available in the bin folder of your Pulsar installation.
+
+</TabItem>
+<TabItem value="REST API">
+
+[REST API](pathname:///admin-rest-api/?version=@pulsar:version_number@): HTTP calls, which are made against the admin APIs provided by brokers. In addition, both the Java admin API and pulsar-admin CLI use the REST API.
+
+</TabItem>
+<TabItem value="Java">
+
+[Java admin API](/api/admin/): it’s a programmable interface written in Java.
+
+</TabItem>
 
-- Go admin API (coming soon)
+</Tabs>
+````
 
-- REST API
+Check the detailed steps below.
 
 ````mdx-code-block
 <Tabs groupId="api-choice"
-  defaultValue="Java admin API"
-  values={[{"label":"Java admin API","value":"Java admin API"},{"label":"REST API","value":"REST API"}]}>
-<TabItem value="Java admin API">
+  defaultValue="pulsar-admin"
+  values={[{"label":"pulsar-admin","value":"pulsar-admin"},{"label":"REST API","value":"REST API"},{"label":"Java","value":"Java"}]}>
+<TabItem value="pulsar-admin">
 
-This tutorial guides you through every step of using Java admin API to manage topics. It includes the following steps:
+This tutorial guides you through every step of using pulsar-admin CLI to manage topics. It includes the following steps:
 
-1. Initiate a Pulsar Java client.
+1. Set the service URL.
 
-2. Create a partitioned topic
+2. Create a partitioned topic.
 
 3. Update the number of a partition.
 
 4. Produce messages to the topic.
 
 5. Check the stats of the topic.
 
-6. Delete the topic.
+6. Delete the topic. 
 
 **Prerequisites**
 
-- Prepare a Java project. You can download one from [Apache Pulsar examples and demos](https://github.com/streamnative/examples).
+- Install and start Pulsar standalone. This tutorial runs Pulsar 2.11 as an example.
 
 **Steps**
 
-1. Initiate a Pulsar Java client in your Java project.
+1. Set the service URLs to point to the broker service in [client.conf](https://github.com/apache/pulsar/blob/master/conf/client.conf).
+
+    ```bash
+    webServiceUrl=http://localhost:8080/
+    brokerServiceUrl=pulsar://localhost:6650/
+    ```
+
+2. Create a persistent topic named test-topic-1 with 6 partitions.
 
     **Input**
 
-    ```java
-    String url = "http://localhost:8080";
-    PulsarAdmin admin = PulsarAdmin.builder()
-        .serviceHttpUrl(url)
-        .build();
+    ```bash
+    bin/pulsar-admin topics create-partitioned-topic \
+    persistent://public/default/test-topic-1 \
+    --partitions 6
     ```
 
-2. Create a partitioned topic _test-topic-1_ with 4 partitions.
+    **Output**
```

Comment:
I would suggest we use the note to tell users there is no output here instead of using **Output** here. same comments for the next step.

Reply:
We do not need to add this kind of "tip" since it's basic knowledge for devs :-D 
Same for Linux. No result is the best result (the operation is successful)